.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "src/LoRaMac-node"]
 	path = src/LoRaMac-node
-	url = https://github.com/Lora-net/LoRaMac-node.git
+	url = https://github.com/boston-engineering/LoRaMac-node.git
```

Comment:
in your fork here can you include output of the linking issue in the git commit log body? have you opened a PR against the upstream repo? we really want to avoid having to use a forked loramac-node if we can avoid it by getting your PR in.

Reply:
done